src/openforms/formio/formatters/tests/test_kitchensink.py

```diff
@@ -12,15 +15,15 @@
 from .utils import load_json
 
 
-def _get_printable_data(submission):
-    printable_data = []
-    for key, (
-        component,
-        value,
-    ) in submission.get_ordered_data_with_component_type().items():
-        printable_data.append(
-            (component["label"], format_value(component, value, as_html=False))
-        )
+def _get_printable_data(submission: Submission) -> list[tuple[str, Any]]:
+    printable_data: list[tuple[str, Any]] = []
+    merged_data = submission.get_merged_data()
+
+    for component in filter_printable(submission.form.iter_components(recursive=True)):
+        key = component["key"]
+        value = format_value(component, merged_data.get(key), as_html=False)
+        printable_data.append((component.get("label", key), value))
+
```

Comment:
This function was already adapted in https://github.com/open-formulieren/open-forms/commit/d7b63bd44d02b6f67efa039dc44d7b2c0453c3fc, and figured it was still relevant as it was kept as well in this commit (while the other uses of `submission.get_printable_data()` were removed).